src/core/trulens/core/utils/serial.py

```diff
@@ -159,6 +158,15 @@ def set(self, obj: Any, val: Any) -> Any:
         """
         raise NotImplementedError()
 
+    def get_sole_item(self, obj: Any) -> Any:
+        all_objects = list(self.get(obj))
+
+        assert (
```

Comment:
is there a reason you're not doing an `if`-`raise` kinda thing? Technically I think all the `assert` can be turned off.

Reply:
did not know this. changed to explicit raise